StepDefinition/LanguageFeatureStepDefinitions.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using OpenQA.Selenium;
+using OpenQA.Selenium.BiDi.Modules.Log;
+using OpenQA.Selenium.Chrome;
+using Project_Mars.Pages;
+using Project_Mars.Utilities;
+using Reqnroll;
+
+
+namespace Project_Mars.StepDefinition
+{
+    [Binding]
+
+    public class LanguageFeatureStepDefinitions : CommonDriver
+    {
+        [Given("I logged into Project Mars successfully for language management")]
+        public void GivenILoggedIntoProjectMarsSuccessfullyForLanguageManagement()
+        {
+            driver = new ChromeDriver();
+
+            LoginPage loginPageObj = new LoginPage();
+            loginPageObj.LoginActions(driver);
+
+            loginPageObj.VerifyUserInHomePage(driver);
+        }
+
+        [When("I navigate to Language")]
+        public void WhenINavigateToLanguage()
+        {
+            HomeToLanguagePage homeToLanguagePageObj = new HomeToLanguagePage();
+            homeToLanguagePageObj.NavigateToLanguage(driver);
+        }
+        [When("I create multiple {string} and {string} in language record")]
+        public void WhenICreateMultipleAndInLanguageRecord(string language, string level)
+        {
+            LanguagePage languagePageObj = new LanguagePage();
```

Comment:
you should make use of constructor this way you do not need to do creations of objects every time             LanguagePage languagePageObj = new LanguagePage();
 Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors